go/cmd/dolt/commands/diff.go

```diff
@@ -103,6 +103,8 @@ In order to filter which diffs are displayed {{.EmphasisLeft}}--where key=value{
 type diffArgs struct {
 	diffParts  diffPart
 	diffOutput diffOutput
+	tableSet   *set.StrSet
+	docSet     *set.StrSet
```

Comment:
Going through this file, all the special casing for docs is pretty painful.  Wish we had a better solution here (Not that it has anything to do with your PR)